src/client/Microsoft.Identity.Client/Cache/Keys/MsalAccessTokenCacheKey.cs

```diff
@@ -78,6 +78,18 @@ public override string ToString()
                 _extraKeyParts);
         }
 
+        public string ToLogString(bool piiEnabled = false)
+        {
+            return MsalCacheKeys.GetCredentialKey(
+                piiEnabled? _homeAccountId : _homeAccountId.GetHashCode().ToString(),
```

Comment:
We should get a stable hash instead. See https://andrewlock.net/why-is-string-gethashcode-different-each-time-i-run-my-program-in-net-core/